src/AsyncMqttClient.cpp

```diff
@@ -378,6 +381,7 @@ void AsyncMqttClient::_addFront(AsyncMqttClientInternals::OutPacket* packet) {
     packet->next = _head;
   }
   _head = packet;
+  _client.setRxTimeout(10);
```

Comment:
Better to move this line to 'connect'. We don't want to set the timeout on every outgoing packet.

Reply:
Oh, and what about using the keepalive value as timeout (seconds!) instead of hardcode to 10 seconds?